piptools/utils.py

```diff
@@ -9,6 +9,14 @@
 from .click import style
 
 UNSAFE_PACKAGES = {"setuptools", "distribute", "pip"}
+COMPILE_EXCLUDE_OPTIONS = {
```

Comment:
any way to extend the click option decorator with a flag for compile_exclude?

Reply:
May be there is a way. Need to research.